pkg/cmd/cli/backup/create.go

```diff
@@ -91,6 +94,8 @@ func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 
 	f = flags.VarPF(&o.IncludeClusterResources, "include-cluster-resources", "", "include cluster-scoped resources in the backup")
 	f.NoOptDefVal = "true"
+
+	flags.BoolVarP(&o.Wait, "wait", "w", o.Wait, "wait for the operation to complete")
```

Comment:
related to the prev comment, we could just move the binding of the `--wait` flag into a different method that only gets called by this command.